clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/handler/KubernetesHandler.java

```diff
@@ -45,7 +45,7 @@
 import java.util.Set;
 
 @Slf4j
-public abstract class KubernetesHandler implements CanDeploy, CanDelete {
+public abstract class KubernetesHandler implements CanDeploy, CanDelete, CanPatch {
```

Comment:
Ah you do have it here -- I would drop it from the other classes since it's not adding anything. `CanDelete` is still around because I never removed it.

Reply:
done!